pkg/release/release.go

```diff
@@ -130,8 +130,8 @@ const (
 	ArchivePath = "archive"
 
 	// Publishing bot issue repository
-	PubBotRepoOrg  = "k8s-release-robot"
-	PubBotRepoName = "sig-release"
+	PubBotRepoOrg  = "kubernetes"
+	PubBotRepoName = "kubernetes"
```

Comment:
I am debating in my mind between possible repositories where this issue should be created. It's the responsibility of the Release Manager to create the rule mappings. Considering that and also the fact that this issue is to signal people to create the rule mapping, the best place to create the issue should be `kubernetes/sig-release` and assign/cc `@kubernetes/release-managers` in the issue body.

WDYT?

Reply:
Sounds good to me. I am not completely aware of the release process. But taking a quick look at the issues in the repository it seems like `k/sig-release` has issues created related to release creation. So that will be more suitable.

Reply:
Done.